source/vir/src/triggers_auto.rs

```diff
@@ -86,7 +98,6 @@ First, we prefer triggers containing the fewest number of terms:
 - {f(x, y)} (1 term) is better (safer) than {g(x), h(y)} (2 terms)
 We choose this because a smaller number of terms leads to fewer quantifier instantiations,
 meaning less chance of an SMT timeout.
-
```

Comment:
I don't know why these three blank lines got removed.  I'd prefer for them to stay.

Reply:
Will do, sorry about that